LIC_FILES_CHKSUM.sha256

```diff
@@ -3,20 +3,27 @@
 32714818ad6f98ee0185a52e23a475d89122e3efd2b2c26c733781c28e798c99  LICENSE
 f42d670262f9aed37a33c97d93fa2c1324f439ee03d7690710c5fe002561b0b5  vendor/github.com/inconshreveable/mousetrap/LICENSE
 5e3400b93bbb099e83e52bab885e7441750673c21f97988ca3f1240639b63283  vendor/github.com/spf13/cobra/LICENSE.txt
-b8514c577c1c4b46cee454d5a882b15fa411e72c5bd7f801f241591789fce61a  vendor/github.com/spf13/pflag/LICENSE
+8514c577c1c4b46cee454d5a882b15fa411e72c5bd7f801f241591789fce61a  vendor/github.com/spf13/pflag/LICENSE
+bf6918bd93ffe07f4b2c61b8287c32cb3122e08aed0be50f1c7d0eddc87877a8f  vendor/gopkg.in/ini.v1/LICENSE
```

Comment:
Smells like a mistake :)

Reply:
I wonder how can the unit tests pass...

Reply:
That is a little worrying, indeed. Looking into it

Reply:
So, @lluiscampos I've dug into this a bit. And the error is that the `shasum` program does not error on this, as it is considered a warning only:
```
shasum: WARNING: 6 lines are improperly formatted
```

These warnings though, stem from the empty lines in the `LIC_FILES_CHECKSUM.sha256` file, which are also considered improperly formatted. 

I will add a fix to `mendertesting next.

Reply:
@oleorhagen Awesome. Thank you for following up

Reply:
Follow up PR: https://github.com/mendersoftware/mendertesting/pull/95/files